tests/test_Voting.py

```diff
@@ -54,32 +75,257 @@ def setup_is_dpytest():
+    vote = cog.vote_manager.get_configuring_vote(guild.members[0].id)
+    assert guild.roles[0].id in vote.target_roles
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote removeRole {guild.roles[0].id}")
+    dpytest.verify_message(f"Vote will no longer be sent to those with the {guild.roles[0].name} role")
+    assert guild.roles[0].id not in vote.target_roles
+
+
+@pytest.mark.asyncio
+async def test_discord_set_chair():
+    config = dpytest.get_config()
+    guild = config.guilds[0]
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote create Test Vote")
+    dpytest.verify_message(
+        f"Vote titled `Test Vote` created for guild {guild.name}. Use {KoalaBot.COMMAND_PREFIX}help vote to see how to configure it.")
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote setChair {guild.members[0].id}")
+    dpytest.verify_message(f"Set chair to {guild.members[0].name}")
     await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote setChair")
     dpytest.verify_message("Results will be sent to the channel vote is closed in")
 
-    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote addOption test option+test1")
-    dpytest.verify_message(f"Option test option with description test1 added to vote")
-    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote addOption test option 2+test2")
-    dpytest.verify_message(f"Option test option 2 with description test2 added to vote")
+@pytest.mark.asyncio
+async def test_discord_add_remove_option():
+    config = dpytest.get_config()
+    guild = config.guilds[0]
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote create Test Vote")
+    dpytest.verify_message(
+        f"Vote titled `Test Vote` created for guild {guild.name}. Use {KoalaBot.COMMAND_PREFIX}help vote to see how to configure it.")
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote addOption test+test")
+    dpytest.verify_message("Option test with description test added to vote")
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote addOption testtest")
+    dpytest.verify_message("Example usage: k!vote addOption option title+option description")
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote removeOption 1")
+    dpytest.verify_message("Option number 1 removed")
+
 
+@pytest.mark.asyncio
+async def test_discord_cancel_vote():
+    config = dpytest.get_config()
+    guild = config.guilds[0]
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote create Test Vote")
+    dpytest.verify_message(
+        f"Vote titled `Test Vote` created for guild {guild.name}. Use {KoalaBot.COMMAND_PREFIX}help vote to see how to configure it.")
+    await dpytest.message(f"{KoalaBot.COMMAND_PREFIX}vote cancel Test Vote")
+    dpytest.verify_message("Vote Test Vote has been cancelled.")
+
+# can't get any tests that involve sending embeds to work
+@pytest.mark.asyncio
+async def test_discord_send_vote():
+    pass
```

Comment:
```suggestion
@pytest.mark.skip(reason="can't get any tests that involve sending embeds to work")
@pytest.mark.asyncio
async def test_discord_send_vote():
    assert False
```
better way to write tests that aren't implemented